src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/libman.json

```diff
@@ -0,0 +1,57 @@
+{
+  "version": "1.0",
+  "defaultProvider": "cdnjs",
+  "libraries": [
+    {
+      "provider": "cdnjs",
```

Comment:
You can remove the `provider` property from each of these object literals. The `defaultProvider` property will be used.